code/GraphQL/LinkablePlugin.php

```diff
@@ -31,6 +32,11 @@ class LinkablePlugin implements ModelQueryPlugin
      */
     private static $field_name = 'link';
 
+    /**
+     * @var array
+     */
+    private static $resolver = [__CLASS__, 'applyLinkFilter'];
```

Comment:
I don't think I've seen a private static written like this before.  Does this even need to be a private static?  Is it something we'd ever expect a project or module to change the configuration for?

Otherwise the old `$query->addResolverAfterware([static::class, 'applyLinkFilter']);` seems appropriate

Reply:
You can pass in a custom resolver through the plugin config. Otherwise, it just falls back on the one that's assigned through static class config. 

You need `__CLASS__` because late static binding (`static::class`) isn't available at the time the static is defined.